roles/web/templates/default.conf.j2

```diff
@@ -0,0 +1,79 @@
+upstream uwsgicluster {
+    {% for host in groups['app'] %}
+    server {{hostvars[host]['ansible_default_ipv4']['address']}}:5000;
+    {% endfor %}
+}
+
+server {
+    listen       80;
+    server_name  {{ansible_fqdn}} www.{{ansible_fqdn}};
```

Comment:
Occurrences of `{{ansible_fqdn}}` here also need to be replaced with `{{inventory_hostname}}`.